corehq/apps/app_manager/tests/test_build_errors.py

```diff
@@ -184,3 +193,29 @@ def test_training_module_as_child(self, *args):
             'type': 'training module child',
             'module': {'id': 1, 'unique_id': 'training_module', 'name': {'en': 'training module'}}
         }, app.validate_app())
+
+    @flag_enabled('DATA_REGISTRY')
+    @patch.object(Application, 'supports_data_registry', lambda: True)
+    def test_multi_select_module_errors(self, *args):
+        factory = AppFactory()
+        module, form = factory.new_basic_module('basic', 'person')
+        factory.form_requires_case(form, 'person')
+
+        module.case_details.short.multi_select = True
+        module.search_config = CaseSearch(
+            search_label=CaseSearchLabel(label={'en': 'Search'}),
+            properties=[CaseSearchProperty(name=field) for field in ['name', 'greatest_fear']],
+            data_registry="so_many_cases",
+            data_registry_workflow=REGISTRY_WORKFLOW_LOAD_CASE,
+        )
+
+        self.assertIn({
+            'type': 'data registry multi select',
+            'module': {'id': 0, 'unique_id': 'basic_module', 'name': {'en': 'basic module'}}
+        }, factory.app.validate_app())
```

Comment:
nit: no grand ideas, but it would be nice if this more clearly was detecting the presence of an error

Reply:
meaning the test itself, like if it were using an `assertRaises`?

Reply:
yeah, or something like `assert_app_has_error(factory.app, error_type, ...)`.  Random musing though, fine to punt

Reply:
ah, interesting - I will punt but keep it in mind